platformio.ini

```diff
@@ -1,13 +1,30 @@
-[env:ttgo-lora32-v1]
+[env]
 platform = espressif32
-board = ttgo-lora32-v1
 framework = arduino
 monitor_speed = 115200
 lib_deps = 
 	adafruit/Adafruit SSD1306@^2.4.2
 	LoRa
 	LoRaLayer2@1.0.1
 build_flags =
-	-Werror
-	-Wall
-	-DARDUINO_LORA 
+	-DARDUINO_LORA
+
+[env:ttgo-lora32-v1]
+board = ttgo-lora32-v1
+build_flags =
+	${env.build_flags}
+	-DMONITOR_CLASS=DisplayMonitor
+	-DLED_TO_BLINK=2
```

Comment:
Should this be set on compile or runtime? A gateway will have a different led pattern from a node?